docs/troubleshoot/troubleshoot-zos-certificate.md

```diff
@@ -1,22 +1,26 @@
-# Troubleshooting the certificate configuration
+# Troubleshooting certificate configuration
 
-The following topic provides the information that can help you troubleshoot problems when you encounter errors or warnings in configuring certificates.
+You may encounter problems when configuring certificates. Review the following article for troubleshoot errors or warnings that can occur when configuring certificates.
 
 ## PKCS12 server keystore generation fails in Java 8 SR7FP15, SR7 FP16, and SR7 FP20
 
 **Symptoms**
 
-When you let Zowe server install create a PKCS12 keystore, the keystore that is generated cannot be read by ZSS. Then, parts of Zowe cannot be used. 
+Some Zowe Desktop applications do not work when Zowe creates a PKCS12 keystore. A message may appear in the log such as the following:
+* ZWES1060W Failed to init TLS environment, rc=1(Handle is not valid)
+* ZWES1065E Failed to configure https server. Check agent https setting.
+
+These messages indicate that ZSS cannot read the generated keystore. As such, parts of Zowe are not functional. 
 
 **Solutions**
 
-This error occurs because of the incompatibility which is found between Java and GSK regarding cryptography.
+This error results from the incompatibility between Java and GSK regarding cryptography.
 
-You can try one of the following options if you are affected by this error.
+Try one of the following options if you are affected by this error:
 
-- Temporarily downgrade Java, for example, to Java 8 SR7FP10.
+- Temporarily downgrade Java, for example to Java 8 SR7FP10, and generate the PKCS12 keystore again.
 
-- Use the flags as below when generating a keystore. 
+- Use the flags as presented in the following codeblock when generating a keystore: 
```

Comment:
Where are these flags supposed to go? I think most users rely on the `zwe init certificates` which reads the `zowe.yaml` but I myself wouldn't know where to configure this.

Reply:
I believe it could be used when you generating the keystore manually using java keytool, it is troubleshooting section so it is fine to have something that not covered by `zwe init certificates` here, it is kind of hotfix on customer side. 
Anyway it all comes from [this issue](https://github.com/zowe/docs-site/issues/2459) and there in discussion Andrej Ch. proposes to use shorter version `-J-Dkeystore.pkcs12.legacy` instead, so we can remove the confusing long flags and keep only legacy flag.